### ISSUE TITLE
Feature request for Sports and additional lines for stocks

### ISSUE THREAD

**Issue by lionsden731** [GeekMagicClock/pixel64x32#1]
Below is an example of how the sports teams look on a different clock. It should be user definable and can handle American football, basketball and baseball. You should also add room for 5 more tickers in the stocks portion for larger portfolios if needed. thanks in advance for taking request
![IMG_0360](https://github.com/user-attachments/assets/604a8100-775d-4385-adc5-594aa4e1d129)
![IMG_0361](https://github.com/user-attachments/assets/3191f3b5-533a-428c-81c1-ffae59f204a0)
![IMG_0362](https://github.com/user-attachments/assets/6656dd8d-c69a-451c-ad59-2c47308bada8)
![IMG_0363](https://github.com/user-attachments/assets/c5ed00d0-8773-4088-9436-db3c15da3321)
![IMG_0364](https://github.com/user-attachments/assets/7c434886-d319-4c5a-a0ce-6fc3fd3dfe23)
![IMG_0367](https://github.com/user-attachments/assets/cb7185ea-7ba1-4957-a513-fa5b24cccfc8)
![IMG_0369](https://github.com/user-attachments/assets/6300d6b3-86be-4c4a-b2be-42c6bbe3b2ba)

**Comment by Moozmanhead** [comment]
Hi,

I also wish to have the possibility to combine tickers with other layouts
like weather, watch and pictures.


Eirik

ons. 4. sep. 2024 kl. 15:57 skrev lionsden731 ***@***.***>:

**Comment by GeekMagicClock** [comment]
@lionsden731  Nice product! Frankly speaking, I don’t know much about sports. I may need some learning. Thank you for sharing. Let me see what we can do.

**Comment by GeekMagicClock** [comment]
Hi, @Moozmanhead , the ticker is already with other layouts, do you mean cycle them in a loop? Or do you have some new layout need to display?

**Comment by lionsden731** [comment]
If you need help on the  sports side let me know. I will be happy to help.

 

Regards,

L

**Comment by GeekMagicClock** [comment]
@lionsden731 Thank you in advance! From which website do you usually get information about these sports events? I want to know if they provide some API interface, preferably free, and preferably include a more comprehensive sports event type. ;)

**Comment by lionsden731** [comment]
> @lionsden731 Thank you in advance! From which website do you usually get information about these sports events? I want to know if they provide some API interface, preferably free, and preferably include a more comprehensive sports event type. ;)

https://developer.yahoo.com/fantasysports/guide/

**Comment by GeekMagicClock** [comment]
@lionsden731  By the way, you mentioned add additional lines for stocks, do you mean, add from 10 stocks to 15 stocks?

**Comment by lionsden731** [comment]
> @lionsden731 By the way, you mentioned add additional lines for stocks, do you mean, add from 10 stocks to 15 stocks?

Yes. I have a few friends that purchased the unit after i recommended it and they ask was it possible to add from 10 to 15 to accommodate their portfolios.

**Comment by GeekMagicClock** [comment]
> > @lionsden731 By the way, you mentioned add additional lines for stocks, do you mean, add from 10 stocks to 15 stocks?
> 
> Yes. I have a few friends that purchased the unit after i recommended it and they ask was it possible to add from 10 to 15 to accommodate their portfolios.

OK, will add this to the next update.

**Comment by Moozmanhead** [comment]
Hi,

Yes I was thinking of cycling them automatically in a loop with other info.

Br

Eirik

tor. 5. sep. 2024, 03:15 skrev GeekMagic ***@***.***>:

**Comment by beenji03** [comment]
Are you able to add a Market Ticker III screen that will display the ticker symbol, price, and percentage change (red negative, green positive) all in the center that scrolls (or marquees) from left to right displaying one asset after another continuously? Perhaps it can start with the current date and time.

**Comment by lionsden731** [comment]
I will check shortly.  ThanksGet Outlook for AndroidFrom: beenji03 ***@***.***>Sent: Wednesday, November 6, 2024 12:57:20 PMTo: GeekMagicClock/pixel64x32 ***@***.***>Cc: lionsden731 ***@***.***>; Mention ***@***.***>Subject: Re: [GeekMagicClock/pixel64x32] Feature request for Sports and additional lines for stocks (Issue #1)
Are you able to add a Market Ticker III screen that will display the ticker symbol, price, and percentage change (red negative, green positive) all in the center that scrolls (or marquees) from left to right displaying one asset after another continuously? Perhaps it can start with the current date and time.
